lib/content-services/src/lib/breadcrumb/breadcrumb.component.spec.ts

```diff
@@ -300,4 +300,26 @@ describe('Breadcrumb', () => {
         expect(component.route[3].id).toBe('test-id');
         expect(component.route[3].name).toBe('test-name');
     });
+
+    it('should set title based on selectedRowItemsCount', () => {
+        component.transform = (transformNode) => {
+            transformNode.id = 'test-id';
+            transformNode.name = 'test-name';
+            return transformNode;
+        };
+        (component.folderNode as any) = {
+            path: { elements: [{ id: 'element-1-id', name: 'element-1-name' }] }
+        };
+        component.ngOnChanges();
+        fixture.detectChanges();
+
+        const titleElement = fixture.debugElement.nativeElement.querySelector('.adf-breadcrumb-item-current');
```

Comment:
You repeat these 2 fragments: 
const titleElement = fixture.debugElement.nativeElement.querySelector('.adf-breadcrumb-item-current');

and 

titleElement.textContent.trim()

because the same two fragments are below. Could you please create some reusable function like (something like getBreadcrumbTitle or something and put these 2 fragments there so then that function can just return this string which you can use for expect?